web/packages/shared/components/UnifiedResources/FilterPanel.tsx

```diff
@@ -332,3 +363,7 @@ const AccessRequestsToggleItem = styled.div`
   text-decoration: none;
   white-space: nowrap;
 `;
+
+function resourceStatusFilterSupported(kinds: string[]) {
+  return !kinds || kinds.length === 0 || !!kinds.find(k => k === 'db');
```

Comment:
```suggestion
function resourceStatusFilterSupported(kinds: string[] = []) {
  return kinds.length === 0 || kinds.includes('db');
```

Reply:
i have to check `!kind` because it gets set to null if no value (default values don't apply to null values)

i tried to change all `null` in `useUrlFiltering` to `undefined` so then the default value will work but noticed that `URLSearchParams.get` returns a null if no value, so now i'm like not sure what to do. i ended up just not changing anything, but let me know